db/schema.rb

```diff
@@ -130,6 +130,8 @@
     t.integer "subject_id"
     t.datetime "created_at", precision: 6
     t.datetime "updated_at", precision: 6
+    t.boolean "main", default: false
+    t.integer "priority"
```

Comment:
I use the name you have here, `priority`, in my comments, but looking around I found [`acts_as_list`](https://github.com/brendon/acts_as_list) which proposes the name `position`. Unfortunately I don't think that that gem will quite work (I don't think it'll work with our model where `position` lies on the join table, not on the item that's being joined), but I think that that name is better than `priority` or `order`.